### ISSUE TITLE
Table Results (Multi-Column)

### ISSUE THREAD

**Issue by dave7777** [running-coder/jquery-typeahead#331]
Hello Tom!
Thank you once again for your wonderful script  - I am trying to replicate the two columns in the results and was wondering if your script can be made to keep the space in the results? Right now the following result:

`AAAA                                       CCC`

is truncated to 

`AAAA CCC`

Dave

**Comment by dave7777** [comment]
Maybe there might be a more elegant way to create two or multi column results - actually that would be table results like here:

https://jsfiddle.net/MrSnrub/pk0kf0hn/

**Comment by running-coder** [comment]
hey dave,

I suggest you use the `template` option to display your results. 

```css
.typeahead__item-column {
  width: 5rem; 
  display:inline-block;
}
```

```javascript
template: "<span class="typeahead__item-column">{{objectKey1}}</span><span class="typeahead__item-column">{{objectKey2}}</span>"
```

**Comment by dave7777** [comment]
Thank you for your kindest reply Tom. I forgot to mention that I am trying to expand your LINK DEMO...http://jsfiddle.net/runningcoder/7e17217d/5/

Do you think it can be made to show the links in the second column?

**Comment by dave7777** [comment]
I was also wondering if your script can be made to keep the space in the results?

**Comment by dave7777** [comment]
Maybe you can create a two column example which woudl have links attached to each row? I think that would lift your scrip from the AWESOME category to the SPECTACULAR ON ALL FRONTS LOL)

**Comment by dave7777** [comment]
this can be closed as it was answered in #329
